flag_engine/segments/models.py

```diff
@@ -56,6 +57,17 @@ def evaluate_not_contains(self, trait_value: typing.Iterable) -> bool:
     def evaluate_regex(self, trait_value: str) -> bool:
         return re.compile(str(self.value)).match(trait_value) is not None
 
+    def evaluate_modulo(self, trait_value: str) -> bool:
+        if type(trait_value) not in (int, float):
```

Comment:
Either this or the type hint is wrong. I assume it's the type hint but worth verifying.

Reply:
This can't be `typing.Union[int, float]` since we are not raising an exception if it's a string, for example? But I feel a bit strange making it `typing.Any or Union[str, int, float, bool]`

Reply:
Yeah, I kinda see your point, but we _are_ expecting it to be any of `Union[str, int, float, bool]` so I think that's the best option.